.github/workflows/arm-auto-signoff-preview.yaml

```diff
@@ -106,4 +108,4 @@ jobs:
         uses: ./.github/actions/add-empty-artifact
         with:
           name: "issue-number"
-          value: ${{ steps.get-label-action.outputs.issueNumber }}
+          value: "${{ fromJson(steps.get-label-action.outputs.result).issueNumber }}"
```

Comment:
Do we need to worry about injection attacks with this?

Reply:
I'm not sure but if someone could attach the issue-number as an artifact that gets picked up by another workflow that does something with it there could be some potential to point at other PRs for example.

Reply:
I considered this and it should be safe, see my comment a few lines above.  If we want to ensure the PR number can always be trusted, we need to either:

1. Extract the PR number from a payload that is trusted, like the event payload of a pull_request trigger.  This cannot be modified by PR authors.

2. Only upload PR number artifacts from "trusted" workflows, meaning a workflow that runs on main (not from the PR branch) and itself extracts the PR number from a trusted context (like the event payload of a workflow_run trigger, where the triggering workflow was itself triggered via pull_request).